Makefile

```diff
@@ -52,6 +56,11 @@ fmt:
 	@test -z "$$(gofmt -s -l . | grep -v vendor/ | tee /dev/stderr)" || \
 		(echo "please format Go code with 'gofmt -s -w'" && false)
 
+.PHONY: lint
+lint: deps
+	## run go lint
+	@test -z "$$(golint ./... | grep -v vendor/ | grep -v ".pb.go:" | grep -vE "context\.Context should be the first parameter of a function" | tee /dev/stderr)"
```

Comment:
This is I think the reason that we can't always enfore this, something `go lint` is too pendantic.